_posts/blog/2018-09-05/2018-09-05-troubleshooting-and-how-to-help-with-development-on-openbazaar.markdown

```diff
@@ -0,0 +1,110 @@
+
+The first thing to do is record the specific error messages you see and/or take screenshots of the problem. **This part is critical for the developers to properly identify the root cause**. You are an archeologist making an archive of this bizarre behavior and you need to ensure it is preserved perfectly. 
+
+* Text errors should always be copied and pasted where possible. 
+* Visual anomalies should be screen captured. 
+* Original file output relevant to the failure should be saved. 
+
+Capturing these artifacts up-front will make sure they don't accidentally get lost by the application changing state automatically or losing power on your computer.
+
+### Finding Solutions
+
+Now that you have the details of the problem, we should make sure that someone else hasn't already reported the problem or found a fix. Often a problem has already been discussed and it's just a matter of using the right tools to find some answers. As a developer on the project, I usually spend 2-3 minutes searching for unique phrases in the error or picking search terms which are going to identify my problem uniquely from other problems. 
+
+For example, consider the following error message:
+
+![OpenBazaar error message indicating error following](There was an error following.png "OpenBazaar error message indicating error following")
+
+The unique parts of this error are **error following** and the phrase **cannot start a transaction within a transaction**. Pay extra attention for developer misspellings if you're transcribing an error from a screengrab, too. While misspellings are embarrassing, they may occur sometimes. Pasting these search phrases into your search engine of choice should reveal pretty quickly if anyone else has reported this issue yet. Making use of double-quotes around phrases which should match exactly is recommended to filter out unrelated results.
+
+![Searching for OpenBazaar error message indicating error following"](Searching for OpenBazaar error following.png "Searching for OpenBazaar error message indicating error following")
+
+
+A search for **OpenBazaar "error following"** produced a Github issue which was reporting the same error. However, when we open the issue we find that the cause of the error is due to something different and probably not related to our problem. I will leave a search for the other phrase as an exercise for you to try. Finding a discussion around an existing problem could lead to a solution that you might be able to apply yourself.
+
+### Researching Resources
+
+For lack of a solution already existing, getting your unreported problem solved will be signficantly more successful if you can get the report into the right hands. We have quite a few places across the internet where the OpenBazaar community offers help to each other. Take advantage of the open community by sharing your challenge in the appropriate channel.
```

Comment:
I'm not sure the for lack of is right here, usually it's "because you don't have X, you did Y". 

I suggest something like "For lack of a solution already existing, you will need to get your unreported problem into the right hands."